### ISSUE TITLE
Study changes to t/op/*.t (8 of 12)

### ISSUE THREAD

**Issue by jkeenan** [atoomic/perl#218]
In this ticket we will study our progress in pursuit of [Objective 2, "strict-by-default"](https://github.com/atoomic/perl/issues/153), for the following files in the Perl core distribution:

```
t/op/or.t
t/op/ord.t
t/op/override.t
t/op/pos.t
t/op/postfixderef.t
t/op/print.t
t/op/push.t
t/op/range.t
t/op/ref.t
t/op/repeat.t
```
These files should be evaluated in light of the guidelines presented on [this wiki page](https://github.com/atoomic/perl/wiki/How-to-Start-Working-with-Tests-in-the-Perl-Core-Distribution).  Please post the summary of your findings in a single comment in this issue.

**First-time contributors:**  (1) Please post a comment to this ticket indicating you are working on it.  That will enable us to assign the ticket to you.  (GitHub won't let us do that until it already thinks you're part of the project.)  (2) You should subscribe to this repository so that you get email updates.

Thank you for helping to improve the Perl programming language.

**Comment by brainbuz** [comment]
I can take a few more of these.

**Comment by brainbuz** [comment]
file | pass | strictures removed | reviewed diffs | notes
-- | -- | -- | -- | --
t/op/or.t | x | x | x |  
t/op/ord.t | x | x | x |  
t/op/override.t | x | x | x |  
t/op/pos.t | x | x | x |  
t/op/postfixderef.t | x | x | x |  
t/op/print.t | x | --- | x | strict line 19
t/op/push.t | x | x | x |  
t/op/range.t | x | x | x |  
t/op/ref.t | x | x | x |  
t/op/repeat.t | x | x | x |

**Comment by jkeenan** [comment]
One corrective p.r. merged into `alpha` in commit d3ea246483.  Thanks for the study.  Closing ticket.

Thank you very much.
Jim Keenan
